src/octotree.js

```diff
@@ -11,8 +11,10 @@ $(document).ready(() => {
   }
 
   function createAdapter() {
-    const githubUrls = store.get(STORE.GHEURLS).split(/\n/).concat('github.com')
-    return ~githubUrls.indexOf(location.host)
+    const githubUrls = store.get(STORE.GHEURLS).split(/\n/)
+      .map((url) => url.replace(/(.*?:\/\/[^/]+)(.*)/, '$1'))
```

Comment:
It does not related with normalize.
When user store with tailing slash, it does not work because `location.host` does not contains tailing slash.